wme-sitebuilder/Wizards/GoLive.php

```diff
@@ -128,6 +179,194 @@ public function verifyDomain() {
 		}
 	}
 
+	/**
+	 * AJAX: search domains.
+	 */
+	public function searchDomains() {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return wp_send_json_error( new WP_Error(
+				'mapps-capabilities-failure',
+				__( 'You do not have permission to perform this action. Please contact a site administrator or log into the Nexcess portal to change the site domain.', 'nexcess-mapps' )
+			), 403 );
+		}
+
+		// Verify the domain structure.
+		$domain = ! empty( $_POST['domain'] )
+			? Domain::parseDomain( $_POST['domain'] )
+			: null;
+
+		$domain = $this->domains->formatDomain( $domain );
+
+		if ( empty( $domain ) ) {
+			return wp_send_json_error( new WP_Error(
+				'mapps-invalid-domain',
+				sprintf(
+					/* Translators: %1$s is the provided domain name. */
+					__( '"%s" is not a valid domain name. Please check your spelling and try again.', 'nexcess-mapps' ),
+					sanitize_text_field( $_POST['domain'] )
+				)
+			), 422 );
+		}
+
+		$response = $this->domains->searchAvailableDomains( $domain );
+
+		if ( is_wp_error( $response ) ) {
+			$companyName = _x( 'Nexcess', 'company name', 'nexcess-mapps' );
+			$companyName = apply_filters( 'nexcess_mapps_branding_company_name', $companyName );
+
+			return wp_send_json_error( new WP_Error(
+				'mapps-search-domains-failure',
+				sprintf(
+					/* Translators: %1$s is the branded company name, %2$s is the API error message. */
+					__( 'The %1$s API returned an error: %2$s', 'nexcess-mapps' ),
+					$companyName,
+					$response->get_error_message()
+				)
+			) );
+		}
+
+		$response_body = wp_remote_retrieve_body( $response );
```

Comment:
We should already be getting formatted data from `searchAvailableDomains`, not a request response.

https://github.com/liquidweb/nexcess-mapps/pull/1477/files#diff-84fbce1c1a4c1d0cea70f924c501345b096ee468c0da7c7b2035ca26c36fb79cR302